### PR TITLE
Update page_10.rst

### PR DESCRIPTION
**Pull request by aditisa1**
Update page_10.rst

### CODE REVIEW

File: source/Pages/page_10.rst [hunk 1]
```diff
@@ -43,14 +43,18 @@ Example to access container logs for modbus-tcp-master container,
...
-Once YML files/docker-compose.yml are changed/Modified in */opt/intel/eii/uwc_data* directory then, execute following command to apply new configurations,
+Once YML files/docker-compose.yml are changed/Modified in /opt/intel/eii/uwc_data directory then, execute following command to apply new configurations,
 
-*sudo ./05_applyConfigChanges.sh*
+DEV Mode:
+sudo ./05_applyConfigChanges.sh
 
-**How to bring up/down the Universal Wellpad Controller containers**
+PROD Mode:
+Refer section 4 "Installation guide". Please execute commands related to following scripts:
```
Comment (kamalraj21intel) [inline source/Pages/page_10.rst:52]: Please update line 52 as follows:
Refer to the [Installation Guide section](https://open-edge-insights.github.io/uwc-docs/Pages/page_04.html). Run the commands related to the following scripts:

File: source/Pages/page_10.rst [hunk 2]
```diff
@@ -43,14 +43,18 @@ Example to access container logs for modbus-tcp-master container,
...
+./02_provision_build_UWC.sh
+./03_Run_UWC.sh
+
+Note: Please download the certificates for internal MQTT broker (if needed).
```
Comment (kamalraj21intel) [inline source/Pages/page_10.rst:56]: Please rewrite this as follows:

Note: If required, download the certificates for the internal MQTT broker.

**Review by kamalraj21intel — COMMENTED** [review]
Please incorporate the review comments @aditisa1

File: source/Pages/page_10.rst [hunk 3]
```diff
@@ -43,14 +43,18 @@ Example to access container logs for modbus-tcp-master container,
...
-Once YML files/docker-compose.yml are changed/Modified in */opt/intel/eii/uwc_data* directory then, execute following command to apply new configurations,
+To apply new configurations after modifying YML files or docker-compose.yml files in the /opt/intel/eii/uwc_data directory, run the following command:
 
-*sudo ./05_applyConfigChanges.sh*
+DEV Mode:
+sudo ./05_applyConfigChanges.sh
 
-**How to bring up/down the Universal Wellpad Controller containers**
+PROD Mode:
+Refer to the Installation Guide section. Run the commands related to the following scripts:
```
Comment (kamalraj21intel) [inline source/Pages/page_10.rst:52]: Please add a hyperlink to the Installation Guide section.

Comment (aditisa1) [reply source/Pages/page_10.rst:52]: Can you please share 
how to do that ?

Comment (kamalraj21intel) [reply source/Pages/page_10.rst:52]: Hi @aditisa1 , Please let me know if you were able to incorporate the changes based on the workaround we discussed in IM. If you still need support then do let me know, we can discuss and close it. Thanks

**Review by kamalraj21intel — APPROVED** [review]

**Pull request merged**